doc/source/examples/incompressible-flow/2d-taylor-couette-flow-nitsche/2d-taylor-couette-flow-nitsche.rst

```diff
@@ -0,0 +1,358 @@
+    end
+  end
+
+
+FEM interpolation
+~~~~~~~~~~~~~~~~~
+
+Lethe supports the use of arbitrary interpolation order. The :math:`\mathcal{L}^2` norm of the error is :math:`\mathcal{O}\left(h^{n+1} \right)` where :math:`h` is a measure of the element size and `n=1` is the interpolation order of the velocity. However, since the torque applied on the inner cylinder depends on the deviatoric stress tensor, which depends on the velocity gradient, it's error will be :math:`\mathcal{O}(n)`. Taking this into account, we use second order polynomials in this example to obtain higher accuracy on the torque. We specify the interpolation order for both pressure and velocity using the ``FEM`` subsection:
+
+.. code-block:: text
+
+    subsection FEM
+        set velocity order            = 1
+        set pressure order            = 1
+    end
+
+.. warning:: 
+    With the ``gls_navier_stokes_2d`` solver, Q2-Q2 elements could also be used. However, we have not found that these lead to better results when the flows are at a low Reynolds number.
+
+
+Analytical solution
+--------------------
+
+Like in the first Taylor-Couette example, we add an analytical solution section to the parameter handler file. This analytical solution is more complex to define, 
+since the simulation domain encompasses the inside of the inner cylinder as well as the gap between the cylinders. Because of this, we only specify the analytical 
+solution for the velocity field and forego pressure. The analytical solution is only defined in the ``.prm`` file and we do not reproduce it here for the sake of brevity.
+
+
+Forces
+-------
+
+The ``forces`` subsection controls the postprocessing of the torque and the forces acting on the boundaries of the domain.
+
+.. code-block:: text
+
+  subsection forces
+    set verbosity             = verbose   # Output force and torques in log 
+    set calculate torques     = true     # Enable torque calculation
+    set torque name           = torque    # Name prefix of torque files
+    set calculation frequency = 1         # Frequency of the force calculation
+    set output frequency      = 1         # Frequency of file update
+  end
+
+
+By setting ``calculate torques=true``, the calculation of the torque resulting from the fluid dynamics physics on every boundary of the domain is automatically calculated. 
+Setting ``verbose=verbose`` will print out the value of the torque calculated for each mesh. 
+
+Simulation control and mesh refinement
+--------------------------------------
+
+As stated above, this problem can either be solved using a uniform mesh refinement or using an adaptative mesh refinement
+
+Uniform mesh refinement
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``simulation control`` subsection controls the flow of the simulation. Two additional parameters are introduced in this example. 
+By setting ``number mesh adapt=4`` we configure to simulation to carry out to solve the fluid dynamics on the mesh and on four(4) subsequently refined mesh. 
```

Comment:
"[...] we configure the simulation to carry out to solve the fluid dynamics [...]"

Reply:
```suggestion
By setting ``number mesh adapt=4`` we configure the simulation to carry out to solve the fluid dynamics on the mesh and on four(4) subsequently refined mesh. 
```